src/cli.ts

```diff
@@ -75,6 +82,7 @@ async function readFileConfig(): Promise<Partial<CliConfig>> {
     const appJson = (await fse.readJson("./app.json")) as AppJson;
     return appJson.svgAppIcon || {};
   } catch (error) {
+    console.log(`Configuration error: ${JSON.stringify(error)}`);
```

Comment:
The tool should work without any configurations, so I don't think we should treat this as a configuration error.